plugins/Monstro/Monstro.cpp

```diff
@@ -503,12 +503,21 @@ void MonstroSynth::renderOutput( fpp_t _frames, sampleFrame * _buf  )
 		if( pd_r > 0.5 ) pd_r = 1.0 - pd_r;
 
 		// multi-wave DC Oscillator
-		len_l = BandLimitedWave::pdToLen( pd_l );
-		len_r = BandLimitedWave::pdToLen( pd_r );
-		if( m_counter2l > 0 ) { len_l /= m_counter2l; m_counter2l--; }
-		if( m_counter2r > 0 ) { len_r /= m_counter2r; m_counter2r--; }
-		sample_t O2L = oscillate( o2w, leftph, len_l );
-		sample_t O2R = oscillate( o2w, rightph, len_r );
+		sample_t O2L = 0.;
+		if (pd_l != 0.)
+		{
+			len_l = BandLimitedWave::pdToLen( pd_l );
+			if( m_counter2l > 0 ) { len_l /= m_counter2l; m_counter2l--; }
+			O2L = oscillate( o2w, leftph, len_l );
+		}
+		
+		sample_t O2R = 0.;
+		if (len_r != 0.)
+		{
+			len_r = BandLimitedWave::pdToLen( pd_r );
+			if( m_counter2r > 0 ) { len_r /= m_counter2r; m_counter2r--; }
+			O2R = oscillate( o2w, rightph, len_r );
+		}
```

Comment:
Same thing here: Should use `typeInfo<float>::isEqual` and could fix formatting as well.

Reply:
Same argument as before against using `typeInfo<float>::isEqual`.